src/components/WwwFrame/Menus/CustomSecondaryMenu.vue

```diff
@@ -0,0 +1,225 @@
+<template>
```

Comment:
This really gets the secondary menu into a more reusable state. At a later time, there may be a way to use this to cleanup the portfolio secondary stuff too.